lib/auth/keystore/aws_kms.go

```diff
@@ -132,16 +141,21 @@ func (a *awsKMSKeystore) generateKey(ctx context.Context, algorithm cryptosuites
 		slog.Any("algorithm", algorithm),
 		slog.Bool("multi_region", a.multiRegionEnabled))
 
+	tags := make([]kmstypes.Tag, len(a.tags))
+	var tagIndex int
+	for k, v := range a.tags {
+		tags[tagIndex] = kmstypes.Tag{
+			TagKey:   aws.String(k),
+			TagValue: aws.String(v),
+		}
+		tagIndex++
```

Comment:
nit: no need to fiddle with indices
```suggestion
	tags := make([]kmstypes.Tag, 0, len(a.tags))
	for k, v := range a.tags {
		tags = append(tags, kmstypes.Tag{
			TagKey:   aws.String(k),
			TagValue: aws.String(v),
		})
```